cmd/kubeadm/app/kubeadm.go

```diff
@@ -34,7 +34,8 @@ func Run() error {
 	pflag.CommandLine.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 
-	pflag.Set("logtostderr", "true")
+	pflag.Set("logtostderr", "false")
+	pflag.Set("alsologtostderr", "true")
```

Comment:
just to double check does this mean that the output would continue to go on STDERR even if the user does not pass --log-file?


this also has to be changed to allow --log-file in combination with --config:
https://github.com/kubernetes/kubernetes/blob/89db31aadefb5207dbf84e2403495d0da31c7c35/cmd/kubeadm/app/apis/kubeadm/validation/validation.go#L539-L571

Reply:
> - does this mean that the output would continue to go on STDERR even if the user does not pass --log-file?
the output would continue to go on STDERR when the user does not pass --log-file
> - logging to a file would only happen if the user passes --log-file, correct?
right
>this also has to be changed to allow --log-file in combination with --config:
Why do we have to allow it?